rpmconf/Install/zmsetup.pl

```diff
@@ -2668,6 +2658,38 @@ sub setAdminPass {
   }
 }
 
+sub setLicenseCode {
+  if ($config{LICENSECODE} eq "") {
+    while (1) {
+      my $new = askNonBlank("Please enter the license code:",$config{LICENSECODE});
+      if (length($new) >= 18 && length($new) <= 24) {
+	      $config{LICENSECODE} = $new;
+	      return;
+      } else {
+	      print "Licensecode should be 18-24 digit!\n";
+      }
+    }
+  }
+}
+
+sub setLicenseActivationChoice {
+  while (1) {
+    my $choice = askNum("Please select one option (1. Activate license with installation 2. Activate license after installation)",$config{LICENSEACTIVATIONCHOICE});
```

Comment:
Please select one option (1. Activate license with installation 2. Activate license after installation)
```suggestion
    my $choice = askNum("Please select one option 1. Activate license with installation 2. Activate license after installation",$config{LICENSEACTIVATIONCHOICE});
```

Reply:
committed suggested changes